build.gradle

```
@@ -1,26 +1,26 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
-    ext.kotlin_version = "1.4.10"
-    repositories {
-        google()
-        jcenter()
-    }
-    dependencies {
-        classpath 'com.android.tools.build:gradle:4.2.0-alpha13'
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+	ext.kotlin_version = "1.4.10"
+	repositories {
+		google()
+		jcenter()
+	}
+	dependencies {
+		classpath 'com.android.tools.build:gradle:4.2.0-alpha14'
```

Comme je viens de mettre à jour Android Studio 4.2.0 à alpha15, du coup cette PR ne compile pas sans mettre à jour les tools 🤦‍♀️ je propose qu'on passe sur alpha15 :D 
```suggestion
		classpath 'com.android.tools.build:gradle:4.2.0-alpha15'
```